### DISCUSSION TITLE
Laserscan sees whole plant including leaves and not only collision body

### DISCUSSION THREAD

**Discussion by moritzboeker** [FieldRobotEvent/Virtual_Field_Robot_Event#27] [Q&A] [answered]
Dear FRE2021-team,

in former versions of your simulation the robot was only able to see the Collision Body (the rod in the center of the plant) of the simulated plants with its laserscan.
I encountered that this has changed and now also the overhanging leaves can be seen within the laserscan. Is this behavior intended? Because now the situation got more difficult as there are also objects within the rows as it can be seen in this picture:
https://ibb.co/BVn0RJc

Sincerely,

Moritz

**Comment by ThijsRuigrok** [comment, marked as answer]
Dear Morits,

Yes, this is the intended behavior. This feature is added on March the 23rd. (https://github.com/FieldRobotEvent/Virtual_Field_Robot_Event/commit/e357246a90e9a1e53de869bc16aab374575eb53f). Making the lidar see the leaves, is more challenging, but also more realistic. 

Kind regards,
Thijs

**Reply by moritzboeker** [reply]
Okay, thank you :-)
